.travis.yml

```diff
@@ -43,6 +43,26 @@ matrix:
     - python: "3.6"
       env: TOXENV=py36-flake8
 
+    # Dashboard JS
+    - language: node_js
+      node_js: "8"
+      cache: npm
+      before_install:
+        - export DISPLAY=:99.0
+        - sh -e /etc/init.d/xvfb start
+        - sleep 3 # give xvfb some time to start
+        - cd ./src/dashboard/frontend/
+      install: npm install
+      before_script:
+        - '' # override global before_script
+      script: npm run "test-single-run"
+      addons:
+        apt:
+          sources:
+            - google-chrome
```

Comment:
Out of curiosity, why is this needed?

Reply:
I was just trying to mimic what we do in the `dashboard.Dockerfile` (download key, add source, etc). But I just simplified it in https://github.com/artefactual/archivematica/pull/1364/commits/36bba916ca5930caa9f2aad13a070048f67dd988 to:

```
addons:
  chrome: stable
```